docs/09-istio.md

```diff
@@ -0,0 +1,78 @@
+# Istio Service Mesh
+While Istio is still in Tech Preview state, it is installable on top of OCP4
+with some small caveats.
+
+## Installing Istio
+Take a look at the [OpenShift 3-based Istio Installation
+Overview](https://docs.openshift.com/container-platform/3.11/servicemesh-install/servicemesh-install.html)
+documentation.
+
+### Skip Node Configuration
+Do not perform the [node
+configuration](https://docs.openshift.com/container-platform/3.11/servicemesh-install/servicemesh-install.html#updating-node-configuration)
+steps. There is a tuned Operator running in OpenShift 4 that will take care
+of this for you.
+
+### Build Your Istio Configuration CR
+The [custom resource
+parameters](https://docs.openshift.com/container-platform/3.11/servicemesh-install/servicemesh-install.html#custom-resource-parameters)
+documentation describes the various options you can put into the CR that tell
+the Operator how to configure Istio. You can also just use the minimal CR to
+start. You can actually find that CR here:
+
+https://raw.githubusercontent.com/Maistra/openshift-ansible/maistra-0.10/istio/istio-installation-minimal.yaml
```

Comment:
We're replacing this in Maistra 0.10 with https://github.com/Maistra/istio-operator/blob/maistra-0.10/deploy/examples/istio_v1alpha1_installation_cr.yaml. Also, this yaml (and the one I just linked) install the community version. Not product.

Reply:
Change the following for a minimal product: 
deployment_type: openshift
  istio:
    community: false

Reply:
Also, this doc is all written against 0.10. 0.10 includes the new operator whereas 0.9 and below use the old operator. The new operator comes with different advised (previous yaml will be deprecated), and all of the docs associated with this will need to be updated.

Reply:
6896ee3 switches to 0.9

we can look to 0.10 for beta4